bindata/network/ovn-kubernetes/self-hosted/ovnkube-master.yaml

```diff
@@ -828,6 +828,12 @@ spec:
             exit 1
           fi
 
+          multi_network_enabled_flag=
+          if [[ "{{.OVN_MULTI_NETWORK_ENABLE}}" == "true" ]]; then
+            multi_network_enabled_flag="--enable-multi-network"
+          fi
+          echo "multi_network_enabled_flag=${multi_network_enabled_flag}"
+
```

Comment:
I think for now we should pass the flag to the binary (instead of the echo) like we do with the gateway_mode_flags.

Reply:
that was my original implementation, in https://github.com/openshift/cluster-network-operator/pull/1699/files#diff-a5b1a97252b229ace4443a7d954a6976d02cc4a6a09b78e3a1c820aaec48fe12

I was told that was redundant, and to align with what it is currently being done for hybrid overlay. 

I really don't care, it can go either way as far as I'm concerned, but seems aligning across all features is not possible.

Reply:
It is true that passing the flag to the binary is redundant as we already have it in the config map.

Similar things could be said about having a meaningless echo on the script though.

We are already redundantly passing config flags for gateway mode, maybe others. I think that from the script standpoint itself is what makes more sense until we have a better way of doing this.

@kyrtapz WDYT?

Reply:
Currently there are several ways we use to trigger the rollout, this needs a refactor across CNO even if we do not want to rollout on every config change. This should be done separately and we should not hold this PR for it.
@maiqueb When I suggested that this flag is redundant I assumed you will get rid of it(together with it's definition) for something else. 

> We are already redundantly passing config flags for gateway mode, maybe others. I think that from the script standpoint itself is what makes more sense until we have a better way of doing this.

Let's go with that for now and we will just include `multi_network_enabled_flag` in the wider refactor in the future.